core/domain/wipeout_service.py

```diff
@@ -1253,3 +1256,85 @@ def _pseudonymize_models_transactional(voiceover_application_models):
             voiceover_application_models[
                 i:i + feconf.MAX_NUMBER_OF_OPS_IN_TRANSACTION]
         )
+
+
+def _pseudonymize_blog_posts_models(pending_deletion_request):
```

Comment:
posts --> post  (since it's models that's pluralized)

Reply:
Done.